### PR TITLE
Upgrade CLImate to latest version

### PR DESCRIPTION
**Pull request by shadowhand**
It doesn't appear the CLImate API has changed significantly. Tests still pass.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -39,7 +39,7 @@
         "php": ">=5.4.0",
         "symfony/process": "~2.0",
         "symfony/console": "~2.0",
-        "league/climate": "~2.0"
+        "league/climate": "~3.0"
```
Comment (keradus) [inline composer.json]: then why not `~2.0|~3.0` ?

Comment (shadowhand) [inline composer.json]: Ah, no reason... I can do that.

File: composer.json [hunk 2]
```diff
@@ -39,7 +39,7 @@
         "php": ">=5.4.0",
         "symfony/process": "~2.0",
         "symfony/console": "~2.0",
-        "league/climate": "~2.0"
+        "league/climate": "~2.0|~3.0"
```
Comment (shadowhand) [inline composer.json]: @keradus changed.

File: .travis.yml [hunk 3]
```diff
@@ -8,6 +8,9 @@ php:
   - hhvm
 
 matrix:
+  include:
+    - php: 5.4
+      env: 'COMPOSER_FLAGS="--prefer-stable --prefer-lowest"'
```
Comment (shadowhand) [inline .travis.yml]: This creates an additional test on Travis for the lowest possible versions of packages, which is good to make sure that BC is maintained.

**Comment by sjparkinson** [comment]
Nice bit of work, thanks.

I'll check and merge in the morning!

**Comment by sjparkinson** [comment]
@shadowhand sorry it took so long to get back to you. I'm actually going to merge #40 shortly.

That failed tests looks to be because of the new `--prefer-lowest` argument. Something up with the symfony/process dependency.

I wouldn't include that change in this PR, rather make a new one so I can work out what's up there and get this merged.

**Comment by shadowhand** [comment]
@homu @sjparkinson rebased and resolved the conflict.

**Comment by shadowhand** [comment]
@sjparkinson it appears that the lowest usable version of `symfony/process` is actually 2.1, I've updated `composer.json` accordingly. This is why I like to have the `--prefer-lowest` check. :smile:

**Comment by sjparkinson** [comment]
Yep thanks @shadowhand, nice catch on the bad version. Looks like 2.0.4 of symfony/process didn't have any autoloading.

**Comment by sjparkinson** [comment]
@homu r+

**Pull request merged**